modules/nw-operator-cr.adoc

```diff
@@ -282,6 +283,14 @@ endif::ibm-z[]
 |`object`
 |Specify a configuration object for customizing network policy audit logging. If unset, the defaults audit log settings are used.
 
+|`gatewayConfig`
+|`object`
+|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway.
+
+[NOTE]
+====
+ While migrating egress traffic, you can expect some disruption to workloads and service traffic until the cluster network operator (CNO) successfully rolls out the changes.
```

Comment:
```suggestion
 While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
```

The Operator is always capitalized: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization.